docs/infrastructure.md

```diff
@@ -0,0 +1,82 @@
+# Infrastructure
+
+## Domains
+
+**nextstrain.org** is [hosted on Heroku](#heroku).
+
+**data.nextstrain.org** is an AWS CloudFronted S3 bucket, [`nextstrain-data`](#nextstrain-data).
+
+**staging.nextstrain.org** is an AWS CloudFronted S3 bucket, [`nextstrain-staging`](#nextstrain-staging).
+
+**login.nextstrain.org** is used by our [AWS Cognito user pool](#cognito).
+
+## Heroku
+
+The production Heroku app is [`nextstrain-server`](https://dashboard.heroku.com/apps/nextstrain-server), which is part of a [Heroku app pipeline of the same name](https://dashboard.heroku.com/pipelines/38f67fc7-d93c-40c6-a182-501da2f89d9d).
+Deploys of `master` happen automatically after Travis CI tests are successful.
+
+A testing/staging app, `nextstrain-dev`, is also used.
```

Comment:
Is this still deployed by pushing to the heroku remote?

Reply:
Yes, it's deployed by pushing to the remote or triggering a build on the Heroku dashboard. Any of the normal Heroku deploy mechanisms work. Automatic deploys of `nextstrain-dev` aren't enabled, because we don't have a dev branch.